enhancements/console/customize-add-page.md

```diff
@@ -0,0 +1,174 @@
+---
+title: customize-add-page
+authors:
+  - "@jerolimov"
+reviewers:
+  - "@spadgett"
+  - "@cvogt"
+  - "@jhadvig"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+creation-date: 2021-02-25
+last-updated: 2021-03-18
+status: implementable
+---
+
+# Customize Add Page
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Motivation
+
+The OpenShift Developer Console provides a long list options to import applications, components and services. Some customers who uses only a part of the import options should hide unused options.
+
+## Summary
+
+Cluster admins want customize the add page of the OpenShift Developer Console.
+
+This list contains a long list of options, some of them depends on installed operators like the OpenShift Pipelines Operator, the OpenShift Serverless Operator, and others.
+
+![Current Add page screenshot](customize-add-page.png)
+
+### Goals
+
+- Add configuration options to allow cluster admins to hide some entries from the add page.
+- Provide a list of available groups and actions so that the admin can hide them.
+
+### Non-Goals
+
+- Introducing a UI to configure this without a yaml editor.
+
+## Proposal
+
+### Extending the Console CRD
+
+Similar to [Customize Developer Catalog Categories](./catalog-categories.md) we extend the existing `operator.openshift.io/v1` / `Console` CRD. It provides already a `spec.customization` area where we can hide groups or actions from the add page.
+
+The cluster admin could disable add groups and actions via its id. This ensures that new later installed operators can show new actions without the requirement to enable them manually.
+
+A final yaml should look like this one:
+
+```yaml
+apiVersion: operator.openshift.io/v1
+kind: Console
+metadata:
+  name: cluster
+  ...
+spec:
+  customization:
+    brand: online
+    addPage:
+      hideGroups:
```

Comment:
I don't think this is needed at this time as per the requirements. Groups are shown based on whether an action is present or not. I'd rather the admin remove the actions only in case new useful actions are added to a group, since we're taking the approach to configuring an exclusion list.

We can always revisit this later and add extra functiionality if required.

Reply:
removed all hideGroups references. Added this as a question with an answer

> No. Groups are shown based on whether an action is present or not. It's fine that the admin needs to disable new actions after a cluster upgrade.